app/presenters/tslr_claim_csv_row.rb

```diff
@@ -36,6 +46,13 @@ def submitted_at
     model.submitted_at.strftime("%d/%m/%Y %H:%M")
   end
 
+  def sanitize_field(field)
+    return field unless field.present?
+
+    field = '="' + field + '"' if DANGEROUS_STRINGS.any? { |string| field.include?(string) }
```

Comment:
I think I'd prefer `"=\"#{field}\""` here. I know it's a bit ugly with the escaped quotes, but it's more performant as concatenating strings like that creates three objects, whereas interpolating strings creates just the one object, which might become an issue with a big CSV.

Reply:
I was trying to avoid the ugly (and hard to read) escaping, but I didn't know about the performance implications of the way Ruby handles this.

Reply:
Could you use `sprintf`? eg. `sprintf('="%s"', field)`

Reply:
Actually standard might complain at that

Reply:
I'm going with the interpolation. I think it's more familiar to read, even with escapes.

Reply:
Done.